spec/components/budgets/investments/ballot_component_spec.rb

```diff
@@ -43,4 +43,44 @@
       expect(page).not_to have_button "Vote", disabled: :all
     end
   end
+
+  describe "price" do
+    let(:budget) { create(:budget, :approval, :balloting) }
+    let(:investment) { create(:budget_investment, :selected, price: 20, budget: budget) }
+    let(:component) do
+      Budgets::Investments::BallotComponent.new(
+        investment: investment,
+        investment_ids: [],
+        ballot: Budget::Ballot.where(budget: budget, user: controller.current_user).first_or_create!
+      )
+    end
+
+    it "is shown price when the budget has not hidey_money active" do
+      sign_in(create(:user, :level_two))
+
+      render_inline component
+
+      expect(page).to have_content "€20"
+    end
+
+    describe "is not shown price when the budget has hidey_money active" do
```

Comment:
```suggestion
    context "when the budget has hidey_money active" do
```

Reasoning: we're still describing the price, but under different conditions, so we use `context`. We mention what the context is about, which is what we do in the `before` block.